stable/artifactory-ha/README.md

```diff
@@ -159,19 +159,28 @@ $ helm install --name artifactory-ha --set artifactory.license.secret=artifactor
 #### Scaling your Artifactory cluster
 A key feature in Artifactory HA ia the ability to set an initial cluster size with `--set artifactory.node.replicaCount=${CLUSTER_SIZE}` and if needed, resize it.
 
+##### Before scaling
+**IMPORTANT:** When scaling, you need to explicitly pass the database password if it's an auto generated one (this is the default with the enclosed PostgreSQL helm chart).
+
+Get the current database password
+```bash
+$ export DB_PASSWORD=$(kubectl get $(kubectl get secret -o name | grep postgresql) -o jsonpath="{.data.postgres-password}" | base64 --decode)
+```
+Use `--set postgresql.postgresPassword=${DB_PASSWORD}` with every scale action to prevent a miss configured cluster!
```

Comment:
You could just use `--resuse-values` instead.

Reply:
I tried this. The `postgresql.postgresPassword` still generates a new value. Maybe because this is an external chart (defined in requirements.yaml)?